api/service_registrations.go

```diff
@@ -0,0 +1,129 @@
+package api
+
+import (
+	"fmt"
+	"net/url"
+)
+
+// ServiceRegistrations is used to query the service endpoints.
+type ServiceRegistrations struct {
+	client *Client
+}
+
+// ServiceRegistration is the fully hydrated object which contains all details
+// of an individual service registration.
+type ServiceRegistration struct {
```

Comment:
Might want to explain the difference between this and `api/services.go#Service`. Something like:

```go
// ServiceRegistrations represent individual allocations registered with a Nomad Service.
// The Services struct represents the service stanza in jobspecs and may become ServiceRegistrations
// in Consul or Nomad depending on provider.
```

...that might be a little too wordy. Just want to try to help people get to what they're looking for as quickly as possible since the names alone don't make it obvious.